vendor/mpdf/mpdf/src/functions.php

```diff
@@ -1,241 +0,0 @@
-<?php
```

Comment:
```
vendor/composer/autoload_files.php
vendor/mpdf/mpdf/src/functions.php
```
These files are deleted, could you add them in a `deleted.files` file? Especially the `functions.php` file is deleted to prevent conflicts with other php libraries.

Reply:
Thank you for pointing this out :+1:
I created this file including all files that have been deleted since, including the previous version of the mpdf library.